.github/workflows/atlas_build-and-push-docker.yml

```
@@ -0,0 +1,197 @@
+name: AtlasML - Build and Push Docker Image natively
+
+on:
+  pull_request:
+    branches: [main]
+    inputs:
```

actionlint (1.7.7)

unexpected key "inputs" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)